device/cuda/src/clusterization/clusterization_algorithm.cu

```diff
@@ -54,7 +60,14 @@ clusterization_algorithm::clusterization_algorithm(
     : m_mr(mr),
       m_copy(copy),
       m_stream(str),
-      m_target_cells_per_partition(target_cells_per_partition) {}
+      m_target_cells_per_partition(target_cells_per_partition),
+      m_f_backup(256 * target_cells_per_partition, m_mr.main),
+      m_gf_backup(256 * target_cells_per_partition, m_mr.main),
+      m_adjc_backup(256 * target_cells_per_partition, m_mr.main),
+      m_adjv_backup(256 * 8 * target_cells_per_partition, m_mr.main),
+      m_backup_mutex(vecmem::make_unique_alloc<unsigned int>(m_mr.main)) {
+    cudaMemset(m_backup_mutex.get(), 0, 1);
```

Comment:
:thinking: Not directly.

I'll have to think about this a little. Since unique pointers don't carry their sizes with them, it's not trivial to create a `vecmem::data::vector_view` from them. :frowning: And I really want to keep `vecmem::copy` operating on views. :thinking: